boards/nxp/rd_rw612_bga/doc/index.rst

```diff
@@ -167,8 +167,36 @@ In this case, the BLE blob won't be linked with the application, so the user nee
 flash the BLE binary blob to the board at the address ``0x18540000``.
 The binary blob will be located here: ``<zephyr workspace>/modules/hal/nxp/zephyr/blobs/rw61x/rw61x_sb_ble_a2.bin``
 
+Board variants
+**************
+
+Ethernet
+========
+
+To use ethernet on the RD_RW612_BGA board, you first need to make the following
+modifications to the board hardware:
+
```

Comment:
Will this be fixed in the next version of the RW612 board?

Reply:
I have no idea what NXP plans are for future versions of the RD board, but on the FRDM board, it doesn't require any modification, and I'm gonna make a PR for that next

Reply:
added a commit for frdm rw612